### PR TITLE
proton transport part of simulation

### PR DESCRIPTION
**Pull request by grzanka**

### CODE REVIEW

File: TotemCondFormats/ProtonTransportFunctions/src/ProtonTransportFunctions.cc [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+ProtonTransportFunctions::~ProtonTransportFunctions()
+{
+//  printf(">> ProtonTransportFunctions::~ProtonTransportFunctions\n");
+}
+
+/*
+//----------------------------------------------------------------------------------------------------
+
+LHCOpticsApproximator* ProtonTransportFunctions::GetFunction(const std::string &name) const
```
Comment (grzanka) [inline TotemCondFormats/ProtonTransportFunctions/src/ProtonTransportFunctions.cc]: Remove

File: TotemCondFormats/DataRecord/interface/ProtonTransportRcd.h [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+//#include "FWCore/Framework/interface/EventSetupRecordImplementation.h"
+#include "FWCore/Framework/interface/DependentRecordImplementation.h"
+#include "TotemCondFormats/DataRecord/interface/BeamOpticsParamsRcd.h"
+#include "Geometry/Records/interface/VeryForwardRealGeometryRecord.h"
+ 
+/**
+ *\brief EventSetup record for proton transport data.
+ **/
+
+//class ProtonTransportRcd : public edm::eventsetup::EventSetupRecordImplementation<ProtonTransportRcd> {};
```
Comment (grzanka) [inline TotemCondFormats/DataRecord/interface/ProtonTransportRcd.h]: Remove

File: TotemCondFormats/BeamOpticsParamsObjects/src/BeamOpticsParams.cc [hunk 3]
```diff
@@ -0,0 +1,421 @@
...
+{
+  char *cmsswPath = getenv("CMSSW_BASE");
+  fileName = std::string(cmsswPath) + "/src/Geometry/TotemRPOptics/data/parametrization_" + GetTagString() + "_reco.root";
+}
+
+
+void BeamOpticsParams::GetStdSimGeometryFile(std::string &fileName) const
+{
+  char *cmsswPath = getenv("CMSSW_BASE");
+  fileName = std::string(cmsswPath) + "/src/Geometry/TotemRPOptics/data/parametrization_" + GetTagString() + "_transp.root";
```
Comment (grzanka) [inline TotemCondFormats/BeamOpticsParamsObjects/src/BeamOpticsParams.cc]: Look, these files are now here:
https://github.com/cms-sw/cmssw/tree/CMSSW_8_1_X/Geometry/VeryForwardProtonTransport/data

File: SimG4Core/Application/src/RunManagerMT.cc [hunk 4]
```diff
@@ -143,6 +145,10 @@ void RunManagerMT::initG4(const DDCompactView *pDD, const MagneticField *pMF,
   // step limiters on top of any Physics Lists
   phys->RegisterPhysics(new ParametrisedEMPhysics("EMoptions",m_pPhysics));
 
+  //Adding Totem proton transport
```
Comment (grzanka) [inline SimG4Core/Application/src/RunManagerMT.cc]: Change comment to "adding fast proton transport to Roman Pot stations"

**Comment by grzanka** [comment]
@ziaber  Does it still work after your changes ?

**Comment by ziaber** [comment]
Yes, if you are asking about BeamOpticsParams.cc change in geometry data directory, it seems that these methods were not used at all.

**Pull request closed without merging**